inc/targetticket.class.php

```diff
@@ -1505,20 +1505,26 @@ public function save(PluginFormcreatorForm_Answer $formanswer) {
 
       // Attach validation message as first ticket followup if validation is required and
       // if is set in ticket target configuration
-      // /!\ Followup is directly saved to the database to avoid double notification on ticket
-      //     creation and add followup
       if ($form->fields['validation_required'] && $this->fields['validation_followup']) {
          $message = addslashes(__('Your form has been accepted by the validator', 'formcreator'));
          if (!empty($formanswer->fields['comment'])) {
             $message.= "\n".addslashes($formanswer->fields['comment']);
          }
 
-        $query = "INSERT INTO `glpi_ticketfollowups` SET
-                     `tickets_id` = $ticketID,
-                     `date`       = NOW(),
-                     `users_id`   = {$_SESSION['glpiID']},
-                     `content`    = \"$message\"";
-         $DB->query($query);
+        // Disable mail notification
```

Comment:
maybe, we could pick up the removed comment (l1508) to explain why we disable notifications.